core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -9174,6 +9174,17 @@ LogicalProject(DEPTNO=[$0], EXPR$1=[$1])
   LogicalSort(sort0=[$2], dir0=[ASC-nulls-first])
     LogicalProject(DEPTNO=[$7], EXPR$1=[RANK() OVER (PARTITION BY $0 ORDER BY $6 DESC NULLS LAST)], EXPR$2=[ROW_NUMBER() OVER (PARTITION BY $0 ORDER BY $6 NULLS FIRST)])
       LogicalTableScan(table=[[CATALOG, SALES, EMP]])
+]]>
+    </Resource>
```

Comment:
Do we need to add this to `SqlToRelConverterTest.xml` if we re not adding anything to `SqlToRelConverterTest.java` ?